src/main/java/step1/Calculator.java

```diff
@@ -0,0 +1,54 @@
+package step1;
+
+public class Calculator {
+	public int result;
+	private final Separator separator;
+
+	public Calculator(String input) {
+		separator = new Separator(input);
+		validCharacter(validNullOrBlank(), "공백, null 은 입력할 수 없습니다");
```

Comment:
```suggestion
		Separator separator = new Separator(input);
		validCharacter(validNullOrBlank(separator), "공백, null 은 입력할 수 없습니다");
		this.separator = separator;
```
클래스의 인스턴스 변수는 검증이 끝난 뒤에 초기화하는 건 어떨까요? 올바르지 않은 값이면 초기화하지 않는 게 더 안전할 것 같아요.